### PR TITLE
Update to v0.11.0-rc.4

### PR DESCRIPTION
**Pull request by tzerrell**
Update all 3 examples to work with v0.11 of risc0.

### CODE REVIEW

File: digital-signature/cli/Cargo.toml [hunk 1]
```diff
@@ -9,9 +9,8 @@ env_logger = "0.9.0"
 log = "0.4.17"
 digital-signature-methods = { path = "../methods" }
 digital-signature-core = { path = "../core" }
-risc0-zkvm-core = "0.9"
-risc0-zkvm-host = "0.9"
-risc0-zkvm-serde = "0.9"
+risc0-zkvm = "0.11.0-rc.4"
+risc0-zkp = { version = "0.11.0-rc.4", default-features = false}
```
Comment (flaub) [inline digital-signature/cli/Cargo.toml:13]: Not sure we need to turn off default-features for the host-side code. But not a problem also.

File: digital-signature/cli/src/lib.rs [hunk 2]
```diff
@@ -63,7 +63,7 @@ pub fn sign(pass_str: impl AsRef<[u8]>, msg_str: impl AsRef<[u8]>) -> Result<Sig
         msg: msg,
     };
 
-    let mut prover = Prover::new(&SIGN_PATH, SIGN_ID)?;
+    let mut prover = Prover::new(&std::fs::read(SIGN_PATH).unwrap(), SIGN_ID).unwrap();
```
Comment (flaub) [inline digital-signature/cli/src/lib.rs:66]: We should prefer to use `Result` than using `unwrap`

**Review by flaub — CHANGES_REQUESTED** [review]
Prefer to avoid the use of `unwrap`

File: digital-signature/cli/src/lib.rs (hunk 2 above)
Comment (tzerrell) [reply digital-signature/cli/src/lib.rs]: The outer `unwrap` here was straightforward to remove. However, I tried to remove the unwrap in `std::fs::read(SIGN_PATH).unwrap()` in a couple of different ways and kept hitting compile errors about "the trait `From<serde::err::Error>` is not implemented for `Exception`" -- I'm not really sure what to do about that.

Comment (tzerrell) [reply digital-signature/cli/src/lib.rs]: similar errors appeared when trying to remove `unwrap`s for `from_slice` and `to_vec`

**Review by flaub — APPROVED** [review]

**Pull request merged**